article/_posts/2020-07-02-What_Matters_in_RL.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: review
+title: "What Matters In On-Policy Reinforcement Learning? A Large-Scale Empirical Study"
+tags: reinforcement deep-learning
+author: "Antoine Théberge"
+cite:
+    authors: "Marcin Andrychowicz, Anton Raichuk, Piotr Stańczyk, Manu Orsini, Sertan Girgin, Raphael Marinier, Léonard Hussenot, Matthieu Geist, Olivier Pietquin, Marcin Michalski, Sylvain Gelly, Olivier Bachem"
+    title:   "What Matters In On-Policy Reinforcement Learning? A Large-Scale Empirical Study"
+    venue:   "Arxiv"
+pdf: "https://arxiv.org/pdf/2006.05990.pdf"
+---
+
+
+# Highlights
+
+- A large scale empirical study on design choices for Reinforcement Learning (RL) algorithms
+- Light recommendations are available from this study
+
+# Introduction
+
+> Deep reinforcement learning (RL) has seen increased interest in recent years due to its ability to have neural-network-based agents learn to act in environments through interactions. For continuous control tasks, on-policy algorithms such as REINFORCE, TRPO, A3C, PPO and off-policy algorithms such as DDPG and SAC have enabled successful applications such as quadrupedal locomotion, self-driving or dexterous in-hand manipulation.
+
+However, many of these algorithms employ high and low-level choices that may not be mentioned in their respective papers. Engstrom et al.[^1] showed that the original PPO paper failed to mention several key design choices that actually make up most of the improvements PPO had over prior methods. Code-level and algorithmic level hidden implementation details severly hinder reproducibility and slow down research in a field where reproducibility is already hard to achieve.[^2] 
+
+# Methods
+
+The authors ran 250 000 experiments to investigate 68 implementation details separated in 8 different groups of choices: Policy Losses, Networks architecture, Normalization and clipping, Advantage Estimation, Training setup, Timesteps handling, Optimizers, and Regularization. The authors selected. For each group, the authors selected which choices might interact the most with each other, and left the remaining choices with default parameters that give competitive results.
```

Comment:
Remove `The authors selected.` I assume it was finally included in the next sentence